episodes/episodes.Rmd

```diff
@@ -1073,13 +1074,120 @@ pie(
 
 ## Math
 
-One of our episodes contains $\LaTeX$ equations when describing how to create
-dynamic reports with {knitr}, so we now use mathjax to describe this:
+One of our episodes contains $\LaTeX$ equations when describing how to create dynamic reports with {knitr}, so we now use mathjax to describe this:
 
 `$\alpha = \dfrac{1}{(1 - \beta)^2}$` becomes: $\alpha = \dfrac{1}{(1 - \beta)^2}$
 
 Cool, right?
 
+
+## Glosario Terms
+
+[Glosario](https://glosario.carpentries.org) is the Carpentries data science glossary that is developed by the international community, providing terms, definitions and translations to make lessons more accessible.
+
+{sandpaper} can automatically generate glossary links in lessons from links to Glosario terms in lesson content markdown.
+
+### Configuration
+
+Firstly, to enable this behaviour, the `glosario` key needs to be added to a lesson's `config.yaml`.
+
+For automatic retrieval of the latest `glossary.yml` from Glosario's GitHub repository, simply set:
+
+```yaml
+glosario: true
+```
+
+For manual retrieval from a local (e.g. if you are working on your lesson offline) or remote location (e.g. providing a specific Glosario version), supply a string relative/absolute path or URL to the config:
+
+```yaml
+# load a glossary from a local file using an absolute path
+glosario: "/path/to/your/glossary.yml"
+
+# load a glossary from a local file using path relative to the lesson
+glosario: "../glossary.yml"
+
+# load a glossary from a remote location
+glosario: "https://foo.com/glossary.yml"
+```
+
+### Lesson content
+
+For example, the [datacarpentry/spreadsheet-ecology-lesson](https://github.com/datacarpentry/spreadsheet-ecology-lesson/blob/main/episodes/01-format-data.md) contains the following markdown:
+
+```
+The most common mistake made is treating spreadsheet programs like lab notebooks, that is, relying on context,
+notes in the margin, spatial layout of data and fields to convey information. As humans, we can (usually) interpret
+these things, but computers don't view information the same way, and unless we explain to the computer what
+every single thing means (and that can be hard!), it will not be able to see how our data fits together.
+
+Using the power of computers, we can manage and analyze data in much more effective and faster ways, but to
+use that power, we have to set up our data for the computer to be able to understand it (and computers are very literal).
```

Comment:
I'm sorry if this is obvious, but I don't understand what this example text is doing here. I expected it to include an example of how Glosario terms are linked, but it doesn't appear to do so.

Reply:
Ah - I see that the version of this text with Glosario links comes later. I don't know that it adds anything to have the non-Glosario'd version of this text here.

Reply:
I was aiming for a "before and after" but if it's clear from the second example with the glosario links included, then I'd be fine removing this!